src/framework/shortcuts/qml/MuseScore/Shortcuts/EditShortcutDialog.qml

```diff
@@ -153,7 +165,10 @@ StyledDialogView {
         }
 
         Keys.onShortcutOverride: function(event) {
```

Comment:
probably we should move this and Keys.onPressed to TextInputField, then we shouldn't activate/deactivate focus for `content`, not in startEdit, not in this function

Reply:
@Eism I tried to move all of this to a TextInputField, but it isn't functioning here. Can you suggest how to implement it here, or should we keep it as it is ?